lib/modules/story_editor/widgets/actor_editor_dialog.dart

```diff
@@ -0,0 +1,96 @@
+import 'package:andax/models/actor.dart';
+import 'package:andax/models/translation_asset.dart';
+import 'package:andax/modules/story_editor/screens/story_editor.dart';
+import 'package:andax/shared/widgets/editor_dialog.dart';
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+
+void showActorEditorDialog(
+  BuildContext context,
+  ValueSetter<Actor?> onResult, [
+  Actor? value,
```

Comment:
I think if `actor == null`, then we shouldn't display the trash icon since there is no actor to delete and the "cancel" button is enough